pools.json

```diff
@@ -1441,5 +1449,6 @@
         "tiger": "tiger",
         "1M1X": "1m1x",
         "OKMiner": "okminer"
+        "Titan": "titan"
```

Comment:
Invalid JSON syntax, you're missing a comma

Reply:
thank you, corrected line in line 1451